sql database/scripts/export.py

```diff
@@ -0,0 +1,98 @@
+# export all images to the db
+USAGE = "python export db_path img_path date"
+
+import sys
+import sqlite3
+from sqlite3 import Error
+import pandas as pd
+import numpy as np
+import json
+
+# create a connection
+def create_connection(db_file):
+	""" create a db connection to database specified
+		by db_file
+	:param db_file: database file path
+	:return: Connection object or None
+	"""
+	try:
+		conn = sqlite3.connect(db_file);
+		return conn
+	except Error as e:
+		print(e)
+
+	return None
+
+# insert image
+def insert_image(conn, image):
+	""" create a new image in the image table
+	:param conn: connection to db
+	:param image: image to insert
+	:return: image name
+	"""
+	sql = ''' INSERT INTO images(Filename,Gtruth,DateCollected,
+	Prediction,Orientation,
+	majorLength,minorLength,Confidence,
+	Height,Width) 
+	VALUES(?,?,?,?,?,?,?,?,?,?) '''
+	cur = conn.cursor()
+	cur.execute(sql, image)
+	return cur.lastrowid
+
+def loadImages(img_path):
+    """	Loads imgs into list of entries
+    :param img_path: Abs path to the database.js
+    :return: list: List of db values
+    """
+    curr_db = ""
+    with open(db_path, "r") as fconv:
+        curr_db = fconv.read()
+
+    db_entries = to_json_format(curr_db)
+    return db_entries
+
+
+def to_json_format(str_db):
+    """	Convert str db into python list of db values (json format)
+    """
+    ind_opbrac = str_db.find("(") + 1
+    str_db = str_db[ind_opbrac:]
+
+    ind_brac = str_db.find(")")
+
+    str_db = str_db[:ind_brac]
+    ind_last_comma = len(str_db) - 2
+    str_db = str_db[:ind_last_comma - 1] + str_db[ind_last_comma:]
+    return json.loads(str_db)
+
+# Execution entrypoint
+if __name__ = '__main__':
+	
+	# check for args
+	if len(sys.argv) <= 3:
+		print(USAGE)
+		exit()
+	# declare cl arguments
+	db = sys.argv[1]
+	img_dat = sys.argv[2]
+	date = sys.arv[3]
+	
+	# get all image files, annotated only for now
+	images = loadDB(img_dat)
+
+	# Store in db
+	# create connection
+	conn = create_connection(db)
+	with conn:
+		# iterate over the images
+		for img in images:
+			
+			filename = img["url"][13:]
```

Comment:
its probably better to string split to get the filename rather than slicing it since these are coming in a formatted manner. Also, probably better to rename `filename` as something like `img_fn`, so its easier to understand since filename is too generic.

Reply:
Please put the whole url as filename without cutting out any strings. So, make it img["url"] instead of img["url"][13:]. Since in /data6/phytoplankton-db/hab_in_vitro/images/20190903/001/00000_static_html/static/images/, there are folders named: 00000  01000  02000  03000  04000, i need the whole url in database.js, which are something like “url”: “images/03000/SPCP-1567533975-004898-005-1200-212-584-336-.jpeg”.

Reply:
Should we come up w/ an adhoc solution to append all of the previous json files with their relative image paths?